### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by martinacantaro**
# TITLE

## Description

Description of the pull request changes and motivation.

## Checklist
- [ ] Linked to Github Issue - there's no related github issue
- [ ] Unit tests added - not needed
- [ ] Integration tests added. - not needed
- [ ] This change requires new documentation. - not needed, this PR is a minor update to the docs
  - [x] Documentation has been added/updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,29 +3,63 @@
...
+
+target/release/cairo-rs-run cairo_programs/abs_value_array_compiled.json --layout all
 ```
 
-## Running a function in a Cairo program with arguments
+
+### Running a function in a Cairo program with arguments
 When running a Cairo program directly using the Cairo-rs repository you would first need to prepare a couple of things. 
 
 1. Specify the cairo program and the function you want to run
```
Comment (fmoletta) [inline README.md:65]: On line 71: `Instantiate the VM, the cairo_runner and the hint processor and the entrypoint`, we could remove that extra `and`

File: README.md [hunk 2]
```diff
@@ -3,29 +3,63 @@
...
-./target/release/cairo-rs-run tests/support/fibonacci_compiled.json
+
+cairo-compile cairo_programs/abs_value_array.cairo --output cairo_programs/abs_value_array_compiled.json
+
+target/release/cairo-rs-run cairo_programs/abs_value_array_compiled.json --layout all
 ```
 
-## Running a function in a Cairo program with arguments
+
+### Running a function in a Cairo program with arguments
```
Comment (fmoletta) [inline README.md:62]: On line 78: There is an extra $ sign in `CairoRunner::new(&$program, "all", false);`, should be just `CairoRunner::new(&program, "all", false);`

Comment (fmoletta) [inline README.md:62]: On line 88: `Lastly, the last thing`

File: README.md [hunk 3]
```diff
@@ -3,29 +3,63 @@
...
 Full compilation and execution example:
 ```bash
 git clone https://github.com/lambdaclass/cairo-rs.git
+
 cd cairo-rs
 
 cargo build --release
-./target/release/cairo-rs-run tests/support/fibonacci_compiled.json
+
+cairo-compile cairo_programs/abs_value_array.cairo --output cairo_programs/abs_value_array_compiled.json
```
Comment (fmoletta) [inline README.md:56]: The `cairo-lang` package is being used here in the example, but it is not listed as a Dependency. We could consider adding it as an optional dependency, or adding a comment about it (ie "Compile the program using `cairo-lang`'s `cairo-compile`

**Review by fmoletta — APPROVED** [review]
Good work! I added some comments, mainly on format/spelling. Most of them address text/code that was not changed by this PR, but an improvement PR is a good opportunity to fix them.

**Review by Juan-M-V — APPROVED** [review]

**Pull request merged**